test/performance/stats.py

```diff
@@ -0,0 +1,21 @@
+import argparse
```

Comment:
I believe we should also have the license header on the python files

```suggestion
# Copyright KubeArchive Authors
# SPDX-License-Identifier: Apache-2.0

import argparse
```

Reply:
Added